spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/batch/listener/EventEmittingItemProcessListener.java

```diff
@@ -64,4 +66,9 @@ else if (item.equals(result)) {
 	public void onProcessError(Object item, Exception e) {
 		messagePublisher.publishWithThrowableHeader("Exception while item was being processed", e.getMessage());
 	}
+
+	@Override
+	public int getOrder() {
+		return 0;
```

Comment:
can you please elaborate more on how you wish this to be implemented . Should i read them one by one in `JobExecutionListenerConfiguration` as they can't be injected into each listener